controllers/articlesEndPointController.js

```diff
@@ -0,0 +1,22 @@
+const { getArticleById } = require("../models/articlesEndPointModel");
+
+const getSinlgeArticle = (req, res, next) => {
+  const articleID = req.params.article_id;
+
+  // if (isNaN(articleID)) {
+  //   return res.status(400).send({ msg: "Bad Request" });
+  // }
+  getArticleById(articleID)
+    .then((article) => {
+      if (!article) {
+        return res.status(404).send({ msg: "Article not found" });
```

Comment:
Just need to remove the commented out code and this old logic on line _11_ and _12_ for `!article` because that's now being handled in the model instead 👌